src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java

```diff
@@ -55,14 +56,12 @@
 
   private static final Logger logger = Logger.getLogger(MainVerticleTest.class);
 
-  private static final String apiKey = "Z1luMHVGdjNMZl9kaWt1X3VzZXI=";
-  private static final String apiKeyForTestUser = "Z1luMHVGdjNMZl90ZXN0X3Rlc3Q=";
-  private static final String badApiKey = "ZnMwMDAwMDAwMA==0000";
+  private static final String API_KEY = "eyJzIjoiajFFWkVaQlI3SSIsInQiOiJkaWt1IiwidSI6InVzZXIifQ==";
+  private static final String API_KEY_FOR_TEST_USER = "Z1luMHVGdjNMZl90ZXN0X3Rlc3Q=";
```

Comment:
pretty sure this would be considered a malformed API key

Reply:
Thanks, Craig. Taken into account.